test/colorization/tle-colorization.test.ts

```diff
@@ -199,16 +199,16 @@ async function assertUnchangedTokens(testPath: string, resultPath: string): Prom
 
                     if (shouldBeArmTemplate) {
                         assert(
-                            testcaseResult.includes('source.json.arm'),
+                            testcaseResult.includes('source.json.arm-template-template'),
                             // tslint:disable-next-line: max-line-length
-                            "This test's filename does not contain '.NOT-ARM.', and so every testcase in it should contain at least one source.json.arm token.");
+                            "This test's filename does not contain '.NOT-ARM.', and so every testcase in it should contain at least one source.json.arm-template-template token.");
                     } else {
                         assert(
-                            !testcaseResult.includes('source.json.arm'),
-                            "This test's filename contains '.NOT-ARM.', but at least one testcase in it contains an source.json.arm token (but shouldn't).");
+                            !testcaseResult.includes('source.json.arm-template-template'),
```

Comment:
Find & replace error? You didn't mean -template-template, did you?

Reply:
Thanks!